package.json

```diff
@@ -29,11 +29,20 @@
     "lint:style": "stylelint '**/*.{,s}css' --ignore-path .gitignore --max-warnings 0",
     "lint:prettier": "prettier . --check --ignore-path .gitignore",
     "test": "jest --coverage",
+    "cy": "concurrently -k yarn:e2e:serve yarn:cypress:open",
```

Comment:
Why did you need to use `concurrently` here?

Reply:
I want to serve the app and run cypress at the same time, in two separate but concurrent processes.

Cypress will wait until the specified url (the app) is ready to serve before running tests.